rllib/algorithms/algorithm_config.py

```diff
@@ -1419,6 +1423,11 @@ def training(
                 dashboard. If you're seeing that the object store is filling up,
                 turn down the number of remote requests in flight, or enable compression
                 in your experiment of timesteps.
+            worker_health_probe_timeout_s: Max amount of time we should spend waiting
+                for health probe calls to finish. Health pings are very cheap, so the
+                default is 1 minute.
+            worker_restore_timeout_s: Max amount of time we should wait to restore
+                states on recovered worker actors. Default is 30 mins.
```

Comment:
I don't think `training()` is a good place for these stuff. Maybe resources? Maybe a new category? e.g. `.fault_tolerance()`?

Reply:
I like the latter better, just for the record :)

Reply:
what do you think about rollout()?

Reply:
how about `.fault_tolerance()` :) ?

Reply:
and instead of worker_xxx let's call them rollout_worker_xxx ? Later we may add the same stuff to trainer workers and I don't want to rename stuff then :) .